sources/packages/web/src/types/contracts/EducationProgram.ts

```diff
@@ -0,0 +1,68 @@
+import { EducationProgramsSummaryAPIOutDTO } from "@/services/http/dto";
+import { OfferingStatus } from "@/types";
+
+/**
+ * Program intensities of the education program.
+ */
+export enum ProgramIntensity {
+  /**
+   * Program with ProgramIntensity = Full Time and Part Time, will be both Full Time and Part Time
+   */
+  fullTimePartTime = "Full Time and Part Time",
```

Comment:
Not a part of the PR, can we change the enum to start with upper case? there were only 3 occurrences in the code.
Also fullTimePartTime can be changed to FullTimeAndPartTime. Just a suggestion.

Reply:
About the upper case, we need to do it globally.
About the `FullTimeAndPartTime`, the way that it is is the way that it should be displayed to the users.